UI/ChatGPT/src/ChatGPT/Services/ChatService.cs

```diff
@@ -43,56 +64,47 @@ public async IAsyncEnumerable<ChatResponse> AskAsStream(ChatRequest chatRequest,
 		var response = new ChatResponse();
 		var content = new StringBuilder();
 
-		IAsyncEnumerator<ChatCompletionCreateResponse>? responseStream = default;
-		while (!response.IsError)
+		AsyncResultCollection<StreamingChatCompletionUpdate> result = _client.CompleteChatStreamingAsync(request);
+		var enumerator = result.GetAsyncEnumerator(ct);
+
+		try
 		{
-			try
+			while (await enumerator.MoveNextAsync() && !response.IsError)
```

Comment:
Same here, the `MoveNextAsync` is no longer in the `try/catch`, it's this part that is the most probable to fail, we need to move it back into it.